### PR TITLE
Add CAP deployment test

### PR DESCRIPTION
**Pull request by kravciak**
Needle: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/997
Local run: http://dhcp126.suse.cz/tests/11206#step/stack_cap/1

Added CAP deployment test before conformance. Test module will:
 - setup nfs server
 - create nfs storage class
 - deploy UAA, SCF
 - login to CF and deploy sample go application

### CODE REVIEW

File: tests/caasp/stack_cap.pm [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    my $mip = script_output q#host -cIN master-api.openqa.test | awk '{print $NF}'#;
+    my $xip = "$mip.xip.io";
+
+    become_root;
+    systemctl 'stop apparmor';
+    systemctl 'stop SuSEfirewall2';
+
+    # Download tools
+    zypper_call 'ar -fG https://download.opensuse.org/repositories/Cloud:/Tools/SLE_12_SP3/Cloud:Tools.repo';
+    zypper_call 'ar -fG https://download.opensuse.org/repositories/openSUSE:/Backports:/SLE-12-SP3/standard/openSUSE:Backports:SLE-12-SP3.repo';
```
Comment (okurz) [inline tests/caasp/stack_cap.pm]: Why do you need unsupported scenarios for a specific SLE version?

Comment (kravciak) [reply tests/caasp/stack_cap.pm]: I need helm & cf-cli tools - we use sles-sp3 support server.
Do you know about supported versions?

File: tests/caasp/stack_cap.pm [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+use utils qw(zypper_call systemctl);
+
+sub run {
+    switch_to 'xterm';
+
+    my $mip = script_output q#host -cIN master-api.openqa.test | awk '{print $NF}'#;
+    my $xip = "$mip.xip.io";
+
+    become_root;
+    systemctl 'stop SuSEfirewall2';
```
Comment (rwx788) [inline tests/caasp/stack_cap.pm]: So am I right that there is no firewalld in any version of caasp, where this test will be used?

Comment (kravciak) [reply tests/caasp/stack_cap.pm]: right, we have sles 12-sp3 support-server and plan is to upgrade to 12-sp4 next year

Comment (rwx788) [reply tests/caasp/stack_cap.pm]: Alright, I personally would write this to have a chance not to touch this code once we upgrade, but it's still a probability =)

**Pull request merged**